packages/math-input/src/components/__tests__/integration.test.tsx

```diff
@@ -80,7 +80,7 @@ function ConnectedMathInput({keypadConfiguration = defaultConfiguration}) {
 }
 
 describe("math input integration", () => {
-    let userEvent;
+    let userEvent: ReturnType<typeof userEventLib.setup>;
```

Comment:
question, nb: So this means whatever the return type is for the `userEventLib.setup` function is the type of this userEvent variable, right?

Reply:
That's exactly correct.